addons/account/models/account_payment.py

```diff
@@ -940,13 +940,16 @@ def _compute_display_name(self):
             payment.display_name = payment.move_id.name if payment.move_id.name != '/' else _('Draft Payment')
 
     def copy(self, default=None):
```

Comment:
you could make us of `copy_data` method instead

Reply:
Hello @MiquelRForgeFlow , 
I dont see the difference here. Also we should return `account.payment` instance but `copy_data` will return a list which we should convert.
Can you please explain further why `copy_data` is better choice here ?

Reply:
The `copy` method, the one in the core, calls `copy_data` to obtain the values that will be used to create the copy. After looking into it, I see that you only are doing the modification of the default, before calling `super`, so it doesn't seem to be any difference between using `copy` or `copy_data`, it will be the same 🤷‍♂️ so sorry for my suggestion.